src/pages/resultsView/enrichments/styles.module.scss

```diff
@@ -18,6 +18,7 @@
   
 .TableContainer {
     width: 800px;
+    margin-right: -10px;
```

Comment:
what does this fix?

Reply:
Not sure but I would think this would just increase the width of the table to 810px. If that's true, then maybe just increase width?